src_c/doc/math_doc.h

```diff
@@ -21,7 +21,7 @@
 #define DOC_VECTOR2ROTATE "rotate(angle) -> Vector2\nrotates a vector by a given angle in degrees."
 #define DOC_VECTOR2ROTATERAD "rotate_rad(angle) -> Vector2\nrotates a vector by a given angle in radians."
 #define DOC_VECTOR2ROTATEIP "rotate_ip(angle) -> None\nrotates the vector by a given angle in degrees in place."
-#define DOC_VECTOR2ROTATEIPRAD "rotate_ip_rad(angle) -> None\nrotates the vector by a given angle in radians in place."
+#define DOC_VECTOR2ROTATEIPRAD "rotate_rad_ip(angle) -> None\nrotates the vector by a given angle in radians in place."
```

Comment:
The doc macro should read `DOC_VECTOR2ROTATERADIP` here :)

PS: These doc headers are autogenerated by running `python3 setup.py docs`, you don't have to manually change these